extensions/machine-learning/src/views/models/manageModels/currentModelsTable.ts

```diff
@@ -284,6 +301,19 @@ export class CurrentModelsTable extends ModelViewBase implements IDataComponent<
 					this.showErrorMessage(`${constants.updateModelFailedError} ${constants.getErrorMessage(error)}`);
 				}
 			});
+			predictButton = this._modelBuilder.button().withProperties({
+				label: '',
+				title: constants.predictModel,
+				iconPath: {
```

Comment:
If you only have one SVG, just use a single path declaration:
iconPath: this.sAbsolutePath('images/makePredictionsSmall.svg')

Reply:
I'm hoping to get another one because this icon doesn't look good in light theme